transport/src/main/java/io/servicefabric/transport/TransportHeaders.java

```diff
@@ -11,6 +11,8 @@
 
   public static final String CORRELATION_ID = "cid";
 
+  public static final String DATA_CLASS = "_class";
```

Comment:
it can be even "class" since it is just string key... I was just thinking to mark with underscore kind of internal system fields (for example it is not recommended to start with "_" application level headers)